package.json

```diff
@@ -46,7 +46,7 @@
     "@readme/eslint-config": "^8.0.2",
     "eslint": "^8.2.0",
     "jest": "^27.0.3",
-    "prettier": "^2.0.1"
+    "prettier": "^2.5.1"
```

Comment:
Can you revert this change and the `package-lock` changes? Looks like you're using npm@6 but we use npm@7. We can handle upgrading `prettier` out of scope of your work here.